docs/en_US/updating/updating-to-14.8-(blobless).md

```diff
@@ -58,7 +57,7 @@ You need a pre-existing jailbreak installed to use this method.
 1. Plug your device into power and connect to the Internet with Wi-Fi
 1. Open the Settings application
 1. Tap `General` -> `Software Update`
-    - Ensure that the version you are updating to is 14.8
+    - Ensure that the version you are updating to is 14.8 **NOT 14.8.1**
```

Comment:
The profile linked is a flat 90-day delay, it won't be 14.8.1 (and it 100% cannot be for those not on iOS versions 14.5-14.8)

Reply:
ahh okay